src/hooks/useSnackbar/useSnackbar.tsx

```diff
@@ -1,76 +1,89 @@
-import { snackbarTransitions } from '@/shared/components/Snackbar'
-import Snackbar from '@/shared/components/Snackbar/Snackbar'
-import { transitions } from '@/shared/theme'
-import { Global } from '@emotion/react'
-import React, { createContext, useContext, useEffect, useState } from 'react'
-import { CSSTransition } from 'react-transition-group'
+import React, { createContext, useContext, useState, useRef, useEffect } from 'react'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+import styled from '@emotion/styled'
+import { Snackbar } from '@/shared/components'
+import { transitions, sizes } from '@/shared/theme'
 
 export type DisplaySnackbarArgs = {
-  time?: number
-  variant?: 'success' | 'error' | 'info'
-  buttonText?: string
+  timeout?: number
+  variant?: 'primary' | 'secondary'
+  icon?: 'success' | 'error' | 'info'
   message: string
+  subMessage?: string
```

Comment:
Sorry for not bringing this up earlier, I think we can improve the naming of those for more clarity. How about `title` for the main one and `message`/`description`/`caption`?